airbyte_cdk/sources/declarative/parsers/model_to_component_factory.py

```diff
@@ -2967,6 +2967,18 @@ def create_simple_retriever(
                 model.requester.request_parameters = self._remove_query_properties(
                     model.requester.request_parameters
                 )
+        elif model.requester.fetch_properties_from_endpoint:
```

Comment:
This does apply a bit of an undocumented behavior where if a person were to define `requester.request_parameters.query_properties` and `requester.fetch_properties_from_endpoint` then we would default to the `request_parameters`

It might make sense to throw an error if both are defined, but right now allowing them to be defined in either places is a little easier and we don't have to go back and modify existing connectors